_posts/2022-09-08-sr2023-registration-open.md

```diff
@@ -27,6 +37,12 @@ Places are limited, so sign up soon to avoid disappointment.
 
 We look forward to seeing your teams!
 
+<div class="text-center">
+<a class="button button-primary" href="/compete#signup">
```

Comment:
Can we include `site.baseurl` here?

Reply:
👍 [95aaba5](https://github.com/srobo/website/pull/431/commits/95aaba507653e041b462c7e62bc0b8a259d85372)